lib/locator.dart

```diff
@@ -1,5 +1,8 @@
 import 'package:get_it/get_it.dart';
+import 'package:idea_tracker/controller/page/create_idea_controller.dart';
 
 GetIt locator = GetIt.instance;
 
-void setupLocator() {}
+void setupLocator() {
+  locator.registerLazySingleton(() => CreateIdeaController());
```

Comment:
Register it as a Factory, that way a new CreateIdeaController is available each time the page is loaded.